### PR TITLE
Add the Big Transfer Model

### PR DESCRIPTION
**Pull request by Shruthi42**
Implements the Big Transfer model from https://arxiv.org/pdf/1912.11370.pdf.
Kolesnikov, Alexander, et al. "Big transfer (BiT): General visual representation learning." arXiv preprint arXiv:1912.11370 (2019).

### CODE REVIEW

**Review by javier-alvarez — APPROVED** [review]

File: InnerEye/ML/models/layers/weight_standardization.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                         kernel_size=kernel_size,
+                         stride=stride,
+                         padding=padding,
+                         dilation=dilation,
+                         groups=groups,
+                         bias=bias,
+                         padding_mode=padding_mode)
+
+    @staticmethod
+    def standardize(weights: torch.Tensor) -> torch.Tensor:
```
Comment (jaysnanavati) [inline InnerEye/ML/models/layers/weight_standardization.py:42]: normalize?

File: InnerEye/ML/models/layers/weight_standardization.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        """
+        Normalize weights on a per-kernel basis for all kernels.
+        """
+        assert weights.ndim == 4  # type: ignore
+        mean = torch.mean(weights, dim=(1, 2, 3), keepdim=True)
+        variance = torch.var(weights, dim=(1, 2, 3), keepdim=True, unbiased=False)
+        standardized_weights = (weights - mean) / torch.sqrt(variance + eps)
```
Comment (jaysnanavati) [inline InnerEye/ML/models/layers/weight_standardization.py:49]: this is doing a z-norm , is there an existing func to do this in torch?

**Review by jaysnanavati — APPROVED** [review]
